lms/static/js/ccx/schedule.js

```diff
@@ -139,7 +139,7 @@ var edx = edx || {};
               return !node.hidden;});
             this.$el.html(schedule_template({chapters: this.showing}));
             $('table.ccx-schedule .sequential,.vertical').hide();
-            $('table.ccx-schedule .toggle-collapse').on('click', this.toggle_collapse);
+            $('table.ccx-schedule .unit a').on('click', this.toggle_collapse);
```

Comment:
For my own information: why change ``.toggle-collapse`` to ``.unit a``? What is the difference?